code/game/objects/items/weapons/weaponry.dm

```diff
@@ -619,3 +619,25 @@
 	force = wielded ? 13 : initial(force)
 	if(user)
 		user.update_inv_hands()
+
+/obj/item/slapper //the active ingredient behind the *slap emote
+	name = "slapper"
+	desc = "This is how real love is shown."
+	icon_state = "latexballon"
+	force = 2
+	throwforce = 0
+	attack_verb = list("slapped")
+	hitsound = 'sound/effects/snap.ogg'
+
+/obj/item/slapper/attack(mob/M, mob/living/carbon/human/user)
+	if(user.a_intent == I_HURT)
+		playsound(M, 'sound/effects/snap.ogg', 50, 1, -1)
+		user.visible_message("<span class='danger'>[user] slaps [M] in the face!</span>",
+ 		"<span class='notice'>You slap [M] in the face! </span>",\
+ 		"You hear a slap.")
+		return
+	else
+		..()
```

Comment:
Wouldn't this be "you hit (target) in the chest with the slapper" or something?

Reply:
yes, it would actually